assets/queries/cloudFormation/elasticache_with_disabled_at_rest_encryption/query.rego

```diff
@@ -0,0 +1,42 @@
+package Cx
+
+
+CxPolicy [ result  ] { 
+
+  document := input.document[i]
+  resource := document.Resources[key]
+  
```

Comment:
At what types of resource does this query check? Type: 'AWS::Route53::RecordSet' and AWS::ElastiCache::ReplicationGroup? If so, we should check for the type to restrict the number of query results